fjord/base/utils.py

```diff
@@ -323,14 +323,30 @@ def ratelimit(rulename, keyfun=actual_ip, rate='5/m'):
         throttling. defaults to actual_ip.
     :arg rate: (optional) rate to throttle at. defaults to 5/m.
 
+    .. Note::
+
+        Unlike django-ratelimit's ratelimit decorator, this does
+        **not** redirect the user or handle the rate limiting.  Caller
+        must check the request for 'limited' and redirect/handle
+        accordingly.
+
+        For example::
+
+            @ratelimit(rulename='foo', keyfun=actual_ip, rate='1/10m')
+            def view(request):
+                if getattr(request, 'limited', False):
+                    # handle ratelimit here!
+                    raise PermissionDenied()
```

Comment:
I added this note so I don't have to go figure out what's going on again. This is one of the big differences between our `ratelimit` decorator and the django-ratelimit one.